src/Maestro/tests/Scenarios/common.ps1

```diff
@@ -32,18 +32,18 @@ if (Test-Path $darcVersion) {
     Write-Host "Using local darc binary $darcTool"
 } else {
     Write-Host "Temporary testing location located at $testRoot"
-    $darcInstallCommand = "dotnet tool install --tool-path $testRoot --version $darcVersion `"Microsoft.DotNet.Darc`""
+    $darcInstallArguments = @( "--tool-path", "$testRoot", "--version", "$darcVersion", "Microsoft.DotNet.Darc" )
```

Comment:
Should need to quote a lot of these, in powershell, `"$example"` is the same as just `$example`